internal/command/command.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package command contains command tests runner.
+package command
+
+import (
+	"context"
+	"errors"
+	"log"
+	"os/exec"
+	"strings"
+
+	"github.com/FerretDB/dance/internal"
+)
+
+// Run runs generic command tests.
+func Run(ctx context.Context, dir string, args []string) (*internal.TestResults, error) {
+	// TODO https://github.com/FerretDB/dance/issues/20
+	_ = ctx
+
+	res := &internal.TestResults{
+		TestResults: make(map[string]internal.TestResult),
+	}
+
+	_ = dir
+	dir = args[len(args)-1]
```

Comment:
Why we ignored passed `dir` and use the last argument instead?

Reply:
To fix the linter, it complaines that the argument dir is overwritten before first use.

Reply:
Well, yeah, the linter is correct. Why we don't use passed dir?